Makefile

```diff
@@ -1,14 +1,9 @@
 REPORTER = $(if $(CI),spec,dot)
 
-ifeq ($(TRAVIS),)
 build: node_modules/
-else
-PATH := $(PWD)/phantomjs/bin:$(PATH)
-build: node_modules/ phantomjs/bin/phantomjs
-endif
 
 test: node_modules/ build lint
-	phantomjs ./node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js \
+	./node_modules/phantomjs-prebuilt/bin/phantomjs ./node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js \
```

Comment:
I was going to suggest that this be `./node_modules/.bin/phantomjs`, but if the `phantomjs-prebuilt` directory is restored from cache, I'm not sure if `.bin/phantomjs` will even exist.